lib/core/cpuMonitor.cpp

```diff
@@ -143,6 +143,12 @@ void CpuMonitor::track_cpu() {
 }
 
 void CpuMonitor::cpu_ult_call_back(connectionInstance& conn) {
+
+    if (!this_thread.joinable()) {
+        conn.send_text_reply("CPU monitor has not been started.");
```

Comment:
I would sent this as a JSON formatted object. That way you don't have to check if the data is parsable or not, just parse as normal, and then look for an error string.